client/src/util/actionHelpers.ts

```diff
@@ -110,13 +116,11 @@ Parameters:
 
 So [1, 2, 3, 4, 10, 11, 14] -> [ [1, 4], [10, 11], 14]
 */
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
 export const rangeEncodeIndices = (
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  indices: any,
+  indices: Array<number>,
   minRangeLength = 3,
   sorted = false
-) => {
+): Array<number | Array<number>> => {
```

Comment:
I believe this is the wrong signature (see the comment on the function).

I believe the return type should be `Array<number | [number, number]>`